lib/marty/monkey.rb

```diff
@@ -95,7 +95,7 @@ class Array
 
 
           def changed_in_place?(raw_old_value, new_value)
-            new_value != type_cast_from_database(raw_old_value)
+            new_value != deserialize(raw_old_value)
```

Comment:
seems a little odd that both type_cast_for_database and type_cast_from_database have been replaced by the same fn deserialize

Reply:
@jockc Good catch, thanks. I guess it should be 'serialize` and `deserialize`. Will check it on weekend

Reply:
Checked it again, and `type_cast_for_database` was never used, so it was present in comment, but the code as it seems instead of serialising new value was deserialising the one from db. Which still happens now. I will just remove redundant comment there